client/src/pages/Profile.js

```diff
@@ -3,15 +3,16 @@ import ProjectCard from '../components/projectCard';
 import ProfileDetailPanel from '../components/ProfileDetailPanel';
 import { withStyles, Button, Grid, Typography } from '@material-ui/core';
 import { getUser } from '../api/users';
+import { withPageContext } from '../components/pageContext';
 
 import './profile.css';
 
 class ProfilePage extends Component {
   state = {
     profile: {
-      id: '',
-      name: 'tony',
-      location: 'orlando',
+      _id: '',
```

Comment:
I actually don't mind the `_id` since it also reminds us we're working with MongoDB. Just my opinion though. I feel like we may be prone to mistakes if we try to make it `id` and somehow we forget to set a `_id` to `id` when we serve up data from our backend.

Reply:
@DevDelvo Ya I can agree, but I've created a sperate icebox ticket that we can decide on one or the other for standard

Reply:
I'm on the anti _id side. I don't think that the front end should have any knowledge or reflection of what's happening on the backend. It is just the object's id right?